addons/payment_stripe/models/payment_acquirer.py

```diff
@@ -27,6 +31,92 @@ class PaymentAcquirer(models.Model):
              "authenticate the messages sent from Stripe to Odoo.",
         groups='base.group_system')
 
+    # === ACTION METHODS === #
+
+    def action_stripe_connect_account(self, menu_id=None):
+        """ Create a Stripe Connect account and redirect the user to the next onboarding step.
+
+        If the acquirer is already enabled, close the current window. Otherwise, generate a Stripe
+        Connect onboarding link and redirect the user to it. If the link generation failed, redirect
+        the user to the acquirer form.
+
+        Note: This method is overridden by the internal module responsible for Stripe Connect.
+        Note: self.ensure_one()
```

Comment:
That note explains nothing, and will format incorrectly when interpreted as rST.